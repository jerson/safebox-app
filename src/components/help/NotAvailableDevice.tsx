import React from 'react';
import { StyleSheet, ViewStyle, View } from 'react-native';
import Text from '../ui/Text';
import Colors from '../../modules/constants/Colors';

const styles = StyleSheet.create({
  container: {
    padding: 10,
    borderRadius: 10,
    backgroundColor: Colors.grey2
  } as ViewStyle,
  notFoundDescription: {
    color: Colors.grey6,
    fontSize: 13
  }
});

function NotAvailableDevice() {
  return (
    <View style={styles.container}>
      <Text style={styles.notFoundDescription}>
        your device does not have any supported method, try another device.
      </Text>
    </View>
  );
}

export default NotAvailableDevice;
