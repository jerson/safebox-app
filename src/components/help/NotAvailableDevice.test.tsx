import React from 'react';
import * as renderer from 'react-test-renderer';
import NotAvailableDevice from './NotAvailableDevice';

const props = {};
test('renders without crashing', () => {
  const snapshot = renderer.create(<NotAvailableDevice {...props} />).toJSON();
  expect(snapshot).toBeTruthy();
});

test('render just views', () => {
  const snapshot = renderer.create(<NotAvailableDevice {...props} />).toJSON();
  expect(snapshot).toMatchSnapshot();
});
