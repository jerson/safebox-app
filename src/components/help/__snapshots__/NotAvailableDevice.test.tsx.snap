// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render just views 1`] = `
<_class
  style={
    Object {
      "backgroundColor": "#f2f4f9",
      "borderRadius": 10,
      "padding": 10,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "fontFamily": "Nunito",
          "fontSize": undefined,
          "fontStyle": "normal",
          "fontWeight": "400",
        },
        Object {},
        Object {
          "color": "#38414e",
          "fontSize": 13,
        },
      ]
    }
  >
    your device does not have any supported method, try another device.
  </Text>
</_class>
`;
